democracy/models/comment.py

```diff
@@ -56,11 +57,25 @@ def parent_id(self):
         """
         return getattr(self, "%s_id" % self.parent_field, None)
 
+    def _detect_lang(self):
+        try:
+            candidates = detect_langs(self.content)
```

Comment:
Noticed that the stupid language detection library is not case-agnostic. Uppercase Finnish get detected as Vietnamese of all things :D Therefore, let's `self.content.lower()` for all content.